downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,17 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### `data.xmldata` packages
+
+- Introduced XML schema definition(XSD) `Sequence` and `Choice` supports for the `data.xmldata` package.
+- Introduced union type support for `xml` operations in the `data.xmldata` package.
+- Introduced finite and enum types support for `xml` operations in the `data.xmldata` package.
```

Comment:
Finite is not a spec term. Use singleton and/or union of singleton appropriately.